src/base/preferences.cpp

```diff
@@ -463,6 +463,16 @@ void Preferences::setServerDomains(const QString &str)
     setValue("Preferences/WebUI/ServerDomains", str);
 }
 
+QString Preferences::getWebUiAddress() const
```

Comment:
@glassez 
Should it return `QString` or `QHostAddress`?

Reply:
I would leave string here.

Reply:
I chose `QString` as `QHostAddress` isn't supported by `QVariant`.

Reply:
OK since it's better for webUI serializing.